### DISCUSSION TITLE
JAR for opentelemetry-exporter-otlp v0.15.0 is empty

### DISCUSSION THREAD

**Discussion by victorpacheco3107** [open-telemetry/opentelemetry-java#2620] [Q&A]
Hi.
I have downloaded the jar opentelemetry-exporter-otlp v0.15.0 from the url [https://mvnrepository.com/artifact/io.opentelemetry/opentelemetry-exporter-otlp/0.15.0](https://mvnrepository.com/artifact/io.opentelemetry/opentelemetry-exporter-otlp/0.15.0) and it is empty.

I noticed this when looking at the libraries in my IDE. Attached image demonstrating this. [https://ibb.co/sjL5LjJ](https://ibb.co/sjL5LjJ)

Thanks.

**Comment by anuraaga** [comment]
Hey - `-otlp` is essentially a pom only artifact, with its dependency on `otlp-trace` being the real deal. I've been meaning to merge these but forgot, will probably do that soon.
